lib/index.js

```diff
@@ -435,20 +435,24 @@ function is_subset(a, b) {
 }
 
 class Hatch {
-    constructor(name, options) {
+    constructor(name, language, options) {
         this._promises = [];
         this._assets = [];
         this._videos = [];
 
-        // If name is not specified, make sure we don't cause problems
-        // but otherwise add it to the prefix
         if (!name) {
-            name = "";
-        } else {
-            name += '_';
+            throw new Error("ERROR! Hatch name must be specified!");
+        }
+
+        // The string check is to ensure that we don't consider older-api `options`
+        // as the language which was in this position before
```

Comment:
Language should be a hard option - we can't pack the shard without it so having it be a part of options (more of an optional map) seems like it would have caused unnecessary pain for ingester writers.